scripts/oss-fuzz.sh

```diff
@@ -57,3 +59,19 @@ $CC $CFLAGS -I"$DESTDIR/usr/include" -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -c -o
 $CXX $CXXFLAGS $LIB_FUZZING_ENGINE secilc-fuzzer.o "$DESTDIR/usr/lib/libsepol.a" -o "$OUT/secilc-fuzzer"
 
 zip -r "$OUT/secilc-fuzzer_seed_corpus.zip" secilc/test
+
+## checkpolicy fuzzer ##
+
+make -C checkpolicy clean
+make -C checkpolicy V=1 CFLAGS="$CFLAGS -I$DESTDIR/usr/include" -j"$(nproc)" checkobjects
+# CFLAGS, CXXFLAGS and LIB_FUZZING_ENGINE have to be split to be accepted by
+# the compiler/linker so they shouldn't be quoted
+# shellcheck disable=SC2086
+$CC $CFLAGS -Icheckpolicy/ -I"$DESTDIR/usr/include" -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -c -o checkpolicy-fuzzer.o checkpolicy/fuzz/checkpolicy-fuzzer.c
+# shellcheck disable=SC2086
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE checkpolicy-fuzzer.o checkpolicy/*.o "$DESTDIR/usr/lib/libsepol.a" -o "$OUT/checkpolicy-fuzzer"
+
+CORPUS_MIN_POL_HASH=$(sha1sum checkpolicy/fuzz/min_pol.mls.conf | awk '{ print $1 }')
```

Comment:
I'm not sure why `sha1sum` is run here. It should be possible to just put both `.conf` files (one of which is currently missing as far as I can see) directly into the seed corpus.

Reply:
https://google.github.io/oss-fuzz/getting-started/new-project-guide/#seed-corpus (emphasis mine):

> To provide a corpus for my_fuzzer, put my_fuzzer_seed_corpus.zip file next to the fuzz target’s binary in $OUT during the build. Individual files in this archive will be used as starting inputs for mutations. **The name of each file in the corpus is the sha1 checksum** (which you can get using the sha1sum or shasum command) of its contents. You can store the corpus next to source files, generate during build or fetch it using curl or any other tool of your choice.

Or do I misread the documentation?

Reply:
> Or do I misread the documentation?

No, you don't. It's true `libfuzzer` itself generates filenames like that but at the same time it has absolutely no problem with reading files named differently. I'd say that the documentation is probably a bit misleading.